bin/setup

```diff
@@ -29,11 +30,10 @@ chdir APP_ROOT do
   # end
 
   puts "\n== Creating development environment configuration variables =="
-  unless File.exist?('.env')
-    system! 'echo "export SECRET_KEY_BASE=$(bin/rails secret)" >> .env'
-    system! 'echo "export SEEDS_ENC_IV=1848E8B20939E8E446A57C9C6745B9E2" >> .env'
-    system! 'echo "export SEEDS_ENC_KEY=" >> .env'
-  end
+  system! 'echo "export GOOGLE_MAPS_API_KEY=" >> .env' if ENV['GOOGLE_MAPS_API_KEY'].nil?
+  system! 'echo "export SECRET_KEY_BASE=$(bin/rails secret)" >> .env' if ENV['SECRET_KEY_BASE'].nil?
+  system! 'echo "export SEEDS_ENC_IV=1848E8B20939E8E446A57C9C6745B9E2" >> .env' if ENV['SEEDS_ENC_IV'].nil?
+  system! 'echo "export SEEDS_ENC_KEY=" >> .env' if ENV['SEEDS_ENC_KEY'].nil?
```

Comment:
This feels a bit cleaner and will keep our `.env` files up-to-date with new variables whenever we run `bin/setup`.